docker/reporter/src/main/java/org/arquillian/cube/docker/impl/client/reporter/DockerLogSection.java

```diff
@@ -0,0 +1,22 @@
+package org.arquillian.cube.docker.impl.client.reporter;
+
+import org.arquillian.reporter.api.event.SectionEvent;
+import org.arquillian.reporter.api.event.TestSuiteSection;
+import org.arquillian.reporter.api.model.report.BasicReport;
+
+public class DockerLogSection extends SectionEvent<DockerLogSection, BasicReport, TestSuiteSection> {
+
+    public DockerLogSection() {
+        super("DockerLog");
+    }
+
+    @Override
+    public TestSuiteSection getParentSectionThisSectionBelongsTo() {
+        return new TestSuiteSection();
```

Comment:
Please, take into account that we could introduce support for multiple test suites in the future. So, make it possible to add the test suite section id - eg. in the same way as it is done in other sections.